signup/style.css

```
@@ -0,0 +1,98 @@
+* {
+	box-sizing: border-box;
+}
+
+:root {
+  --light-blue-color: #E6F2FF;
+  --blue-color: #3692FF;
+  --gray900-color: #111827;
+  --gray800-color: #1F2937;
+  --gray700-color: #374151;
+  --gray600-color: #4B5563; 
+  --gray500-color: #6B7280; 
+  --gray400-color: #9CA3AF;
+  --gray200-color: #E5E7EB;
+  --gray100-color: #F3F4F6;
+  --gray50-color: #F9FAFB;
+
+}
+
+body {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+header {
+  margin-top: 50px;
+}
+
+form {
+  margin: 16px auto;
+  width: 640px;
+}
+
+label {
+  color: #2c2c2c;
+  display: block;
+  margin: 16px 0 8px;
+  font-weight: bold;
+}
+
+input {
+  background: var(--gray100-color);
+  border: none;
+  border-radius: 10px;
+  display: block;
+  font-size: 16px;
+  line-height: 24px;
+  margin: 8px 0;
+  padding: 16px 24px;
+  width: 100%;
+}
+
+input[type=password] {
+  background-image: url('/images/icon/ic_visible_off.png');
+  background-position: top 16px right 16px;
+  background-repeat: no-repeat;
+}
+
+input:focus {
+  outline: none !important;
+  border: 1px solid var(--blue-color);
```

outline을 없애기 위해 `!important`를 사용하는 것은 지양하는 것이 좋습니다.
[참고 링크](https://velog.io/@joahkim/CSS-%EC%84%A0%ED%83%9D%EC%9E%90-%EC%9A%B0%EC%84%A0%EC%88%9C%EC%9C%84)에서 설명 되어있는 것처럼 CSS 우선순위에 방해가 되고 장기적으로는 !important가 남발되는 상황이 생길 수 있습니다.

---

```suggestion
  outline: 1px solid var(--blue-color);
```
그리고 border 보다는 outline 자체에서 효과를 주는 것을 추천드립니다. 현재는 border 크기인 1px 만큼 아래 요소가 밀려나는 현상이 발생하고 있습니다. 이렇게 레이아웃시프트가 발생하게 되면 다른 요소들이 재배치되면서 성능에 악영향을 줄 수 있어서요!